contrib/terraform/openstack/variables.tf

```diff
@@ -294,10 +294,47 @@ variable "router_internal_port_id" {
 
 variable "k8s_masters" {
   default = {}
+  type = map(object({
+    az                     = string
+    flavor                 = string
+    floating_ip            = bool
+    etcd                   = bool
+    image_id               = optional(string)
+    root_volume_size_in_gb = optional(number)
+    volume_type            = optional(string)
+    network_id             = optional(string)
+  }))
```

Comment:
Do we need to define the same type here and contrib/terraform/openstack/modules/compute/variables.tf ?
It would be helpful if we can reduce the places of the type definitions for reducing our maintenance burden.

Reply:
You are correct, that's super unnecessary. I don't know why I assumed that you needed them at both places.